### PR TITLE
fix issues in DomainExtract

### PR DESCRIPTION
**Pull request by meirwah**

### CODE REVIEW

File: Scripts/script-ExtractDomain.yml [hunk 1]
```diff
@@ -40,6 +40,18 @@ script: |-
       return domain;
   };
 
+  var scoreToReputation = function(score) {
```
Comment (aviadl) [inline Scripts/script-ExtractDomain.yml]: Why not in common ?

File: Scripts/script-ExtractDomain.yml [hunk 2]
```diff
@@ -75,13 +87,30 @@ script: |-
...
+      var hostname = extractHostname(uniqueMatches[i]);
+      var rootDomain = extractRootDomain(uniqueMatches[i]);
+      var domains = [];
+      domains.push(rootDomain);
+      if (domain != hostname) {
+          domains.push(hostname);
+      }
+      for (var j=0; j < domains.length; j++) {
+          var domain = domains[j];
+          var res = executeCommand("findIndicators", {"query": "type:Domain* and value:\"" + domain + "\""});
```
Comment (aviadl) [inline Scripts/script-ExtractDomain.yml]: better to ask before the search if to mark as indicator

File: Scripts/script-ExtractDomain.yml [hunk 3]
```diff
@@ -75,13 +87,30 @@ script: |-
...
+          if (!(res && res[0] && res[0].Contents[0]) && markAsIndicator) {
```
Comment (aviadl) [inline Scripts/script-ExtractDomain.yml]: maybe add a param to ask if to calculate reputation or not, with default to true ?

**Comment by meirwah** [comment]
@aviadl can you approve?

**Pull request merged**